features/support/hooks.rb

```diff
@@ -6,3 +6,13 @@
   Capybara.reset_sessions!
   Capybara.current_driver = :rack_test
 end
+
+
+Before('@poltergeist') do
```

Comment:
It is a duplicate.  It was handled in #127 .  Not sure why it wasn't changed when I merged with a current pull of develop.  Hm.

I'll remove the duplication.

good catch.